### PR TITLE
NOTES: change port-forward to single command

### PR DESCRIPTION
**Pull request by inodb**
**What this PR does / why we need it**:
Change NOTES to forward the port of the service instead of pod which is a bit more readable

### CODE REVIEW

**Comment by davidkarlsen** [comment]
@inodb Chart.yaml needs version bump. Also can you sign the CLA?

File: stable/mysql/templates/NOTES.txt [hunk 1]
```diff
@@ -34,9 +34,8 @@ To connect to your database directly from outside the K8s cluster:
     MYSQL_HOST=127.0.0.1
     MYSQL_PORT={{ default "3306" .Values.service.port }}
 
-    # Execute the following commands to route the connection:
-    export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "mysql.fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
-    kubectl port-forward $POD_NAME {{ default "3306" .Values.service.port }}:{{ default "3306" .Values.service.port }}
+    # Execute the following command to route the connection:
+    kubectl port-forward svc/{{ template "mysql.fullname" . }} {{ default "3306" .Values.service.port }}:{{ default "3306" .Values.service.port }}
```
Comment (davidkarlsen) [inline stable/mysql/templates/NOTES.txt]: is the default needed, see: https://github.com/helm/charts/blob/master/stable/mysql/templates/svc.yaml - it uses .Values.service.port

**Comment by inodb** [comment]
@davidkarlsen thanks for reviewing - bumped version and removed 3306 port defaults

File: stable/mysql/templates/NOTES.txt [hunk 2]
```diff
@@ -32,11 +32,10 @@ To connect to your database directly from outside the K8s cluster:
...
     {{- else if contains "ClusterIP" .Values.service.type }}
     MYSQL_HOST=127.0.0.1
-    MYSQL_PORT={{ default "3306" .Values.service.port }}
+    MYSQL_PORT={{ .Values.service.port }}
 
-    # Execute the following commands to route the connection:
-    export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "mysql.fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
-    kubectl port-forward $POD_NAME {{ default "3306" .Values.service.port }}:{{ default "3306" .Values.service.port }}
+    # Execute the following command to route the connection:
+    kubectl port-forward svc/{{ template "mysql.fullname" . }} {{ .Values.service.port }}:{{ .Values.service.port }}
```
Comment (davidkarlsen) [inline stable/mysql/templates/NOTES.txt]: .Values.service.port is double up

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/mysql/templates/NOTES.txt (hunk 2 above)
Comment (inodb) [reply stable/mysql/templates/NOTES.txt]: fixed

Comment (davidkarlsen) [reply stable/mysql/templates/NOTES.txt]: ah - silly me - that wasn't really wrong - but anyways - it looks good either way

**Pull request merged**